src/libraries/System.Runtime/tests/System/HalfTests.cs

```diff
@@ -991,16 +1011,16 @@ public static IEnumerable<object[]> ToStringRoundtrip_TestData()
             yield return new object[] { Half.MaxValue };
             yield return new object[] { Half.PositiveInfinity };
 
-            yield return new object[] { (UInt16BitsToHalf(0b0_00001_0000000000))}; // smallest normal
-            yield return new object[] { (UInt16BitsToHalf(0b0_00000_1111111111))}; // largest subnormal
-            yield return new object[] { (UInt16BitsToHalf(0b0_00000_1000000000))}; // middle subnormal
-            yield return new object[] { (UInt16BitsToHalf(0b0_00000_0111111111))}; // just below middle subnormal
-            yield return new object[] { (UInt16BitsToHalf(0b0_00000_0000000001))}; // smallest subnormal
-            yield return new object[] { (UInt16BitsToHalf(0b1_00000_0000000001))}; // highest negative subnormal
-            yield return new object[] { (UInt16BitsToHalf(0b1_00000_0111111111))}; // just above negative middle subnormal
-            yield return new object[] { (UInt16BitsToHalf(0b1_00000_1000000000))}; // negative middle subnormal
-            yield return new object[] { (UInt16BitsToHalf(0b1_00000_1111111111))}; // lowest negative subnormal
-            yield return new object[] { (UInt16BitsToHalf(0b1_00001_0000000000))}; // highest negative normal
+            yield return new object[] { (UInt16BitsToHalf(0b0_00001_0000000000)) }; // smallest normal
```

Comment:
Ugh this sneaked through. Let me know if you want me to remove it.

Reply:
I think it's fine, it's just normalizing spacing 😄